Jenkinsfile

```diff
@@ -38,6 +38,6 @@
  */
 // To use a test branch (i.e. PR) until it lands to master
 // I.e. for testing library changes
-//@Library(value="pipeline-lib@your_branch") _
+@Library(value="pipeline-lib@bmurrell/build-and-test-on-leap15.3") _
```

Comment:
this needs to be commented out again?

Reply:
Yeah.  I had actually removed you from review as quickly as I accidentally added you.  Clicked on the wrong PR.  This one is not quite ready yet.  Apologies.